src/Simulations/run.jl

```diff
@@ -168,20 +165,21 @@ function run!(sim; pickup=false, callbacks=[])
         for n in 1:iterations
             clock.time >= sim.stop_time && break
 
-            # Temporary fix for https://github.com/CliMA/Oceananigans.jl/issues/1280
             aligned_Δt = aligned_time_step(sim)
+
+            # Temporary fix for https://github.com/CliMA/Oceananigans.jl/issues/1280
```

Comment:
Is this an outstanding issue that we want to fix?  Should not be here but wondering if it's on our radar.

Reply:
I'm not sure. I don't see a different way of achieving time-step alignment.